lib/app.js

```diff
@@ -0,0 +1,49 @@
+require('dotenv').config();
+require('./utils/connect')();
+const Drink = require('./models/Drink');
+const express = require('express');
+const app = express();
+const mongoose = require('mongoose');
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
+  res.send({ drinks: ['coffee', 'orange juice', 'milk', 'beer', 'cider', 'wine'] });
+});
+
```

Comment:
missing a findById route

Reply:
Ah, my bad! I'll fix that today and resubmit